Headers/DebugServer2/Core/HardwareBreakpointManager.h

```diff
@@ -61,6 +61,10 @@ class HardwareBreakpointManager : public BreakpointManager {
                    std::function<void(Target::Thread *t)> const &cb) const;
```

Comment:
I think the title of this commit is wrong. You're adding a wrapper for *reading* debug registers, no?